### PR TITLE
Add guide on how to contribute to this project

### PR DESCRIPTION
**Pull request by Cruikshanks**
Obviously, this is a fantastic resource for anyone new joining the team 💪! But its been suggested they might identify ways it can be improved 😱

So, just in case this _unlikely_ event happens this change adds a guide on how to contribute issues and changes.

### CODE REVIEW

**Comment by Jozzey** [comment]
Looks good & makes sense. Found the deliberate mistake ;-)

File: contributing.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+> Don't panic though! If you are still unsure just crack on with your contribution. You can't break anything hear and we'd rather see your idea than miss out because we didn't write this clearly enough.
+
+## Guides and documents
+
+Any suggested change to the project's contents, by which we mean the files and folders, should be done using a PR. Have a read of [ways of working](/ways_of_working.md) first. This is because the format for commits and the way we process PR's here is the same as for any of our code repos.
+
+Assign it a [label](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels) but again don't panic about correctness. Some suggestions would be
+
+- correcting a typo or incorrect fact? `bug`
```
Comment (jonathangoulding) [inline contributing.md:13]: I would suggest this be automated: 

Using conventional commits - https://github.com/action-runner/conventional-labeler

And this action for folder / file changes - https://github.com/actions/labeler (could be useful for the cypress test changes in the UI repo)

Comment (jonathangoulding) [reply contributing.md:13]: https://www.conventionalcommits.org/en/v1.0.0/

File: contributing.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+- check again
+- go away, have a goodnights sleep, then check again!
+
+Seriously, issue lists have a habit of getting out of hand especially on a project with as much to fix as **Water Abstractions**. So the list doesn't become unmanageble do please check your not duplicating something, or whether you could instead be adding to an existing one.
+
+Other than that you are good to go. Don't restrict yourself to just problems that need fixing. If there is something new we could be doing, either in code or our processes then feel free to use issues as a why of collecting initial feedback.
+
+### Progressing the issue
+
+Unlike changes via PR there is no formal approval process. But before you immediately dive into, for example, implementing TypeScript, do double-check with your team mates this is something they agree with.
```
Comment (jonathangoulding) [inline contributing.md:41]: Adding a link the GitHub description adds a mentions link the issue. 

Maybe have this as part of the process ? 

Screenshot 2022-05-16 at 09 10 07

**Review by jonathangoulding — COMMENTED** [review]
Some comments

File: contributing.md (hunk 1 above)
Comment (Cruikshanks) [reply contributing.md:13]: TIL about conventional commits and why @jonathangoulding 's commits start with `chore:` and `feat:`! 😂 

However, (and this is IMHO), I think we have a lot going on right now so I don't want to complicate our processes further. This applies to what we do and our workflows.

It's purely subjective; I think asking for a _fuller_ commit message that isn't just a link to the JIRA ticket is doable. Asking folks to follow a specific spec might be a step too far right now 🤷 .

[Pull Request Labeler](https://github.com/actions/labeler) is definitely worth look but

- the context for this is labelling an issue not a PR. So, not sure it'd work in this context
- I want to get a handle on the CI pipeline across the reps are complete the other issues related to that before I think of then expanding functionality again

So, right now this goes on my backlog for things to revisit once we've (I've!) got more of a handle on this service.

Hope that's ok

File: contributing.md (hunk 2 above)
Comment (Cruikshanks) [reply contributing.md:41]: That makes perfect sense. Guess I assumed folks would link to the issue in their PRs but sure won't hurt to state it.

Comment (Cruikshanks) [reply contributing.md:41]: Actually, I did mention it in there

> If the issue results in changes in other repos link to the issue in your commits. GitHub is clever enough to show them in the issue.

But I've tidied it up and added your image as a nice example.

**Pull request merged**